src/main/java/com/microsoft/spring/data/gremlin/conversion/script/GremlinScript.java

```diff
@@ -13,9 +13,25 @@
  */
 public interface GremlinScript<T> {
     /**
-     * Generate the final script before submit query to gremlin server.
+     * Generate the final insert script query to gremlin server.
```

Comment:
1. From this comment, I don't see more information than method name. So maybe rephrase it,
2. Returning generic type here make it difficult to understand what's the real return type. If String is only possible return type, no need to use generic.